sdk/servicebus/service-bus/src/receiver.ts

```diff
@@ -125,6 +125,17 @@ export class Receiver {
     }
   }
 
+  /**
+   * Gets an async iterator over messages from the receiver.
+   * The iterator exits if it is not able to fetch a new message in over a minute.
+   */
+  async *getMessageIterator(): AsyncIterableIterator<ServiceBusMessage> {
+    while (true) {
+      const currentBatch = await this.receiveMessages(1);
+      yield* currentBatch;
```

Comment:
`yield currentBatch[0]` generates significantly less code so should be preferred here I think.

Reply:
done